src/battle_ai_util.c

```diff
@@ -57,14 +57,15 @@ bool32 BattlerHasAi(u32 battlerId)
     switch (GetBattlerPosition(battlerId))
     {
     case B_POSITION_PLAYER_LEFT:
-        if (IsAiVsAiBattle())
+        if (AI_THINKING_STRUCT->aiFlags[battlerId] & AI_FLAG_OMNISCIENT && IsAiVsAiBattle())
```

Comment:
this is saying the player left slot has _any_ AI only if the omniscient flag is set

Reply:
Oh, oops.

Reply:
Hm, I will have to rethink that. Can I remove this specific check and keep the other one since it is still useful?

Reply:
@ghoulslash could you confirm that I can remove the check for now to make it ready for a merge? :eyes: